wolfssl/wolfcrypt/wc_port.h

```diff
@@ -187,15 +337,19 @@
     #define lkm_printf(format, args...) printk(KERN_INFO "wolfssl: %s(): " format, __func__, ## args)
     #define printf(...) lkm_printf(__VA_ARGS__)
 
+    #ifdef HAVE_FIPS
```

Comment:
Ideally the FIPS headers would do this when `NO_ATTRIBUTE_CONSTRUCTOR` is defined. Again, just a note... no action required.

Reply:
huh, yeah it seems weird that I needed that.  I tried again to remove it just a few minutes before pushing and still got the missing-declaration warning despite the ```    #define NO_THREAD_LS
    #define NO_ATTRIBUTE_CONSTRUCTOR
``` added in this PR.

digging further, I see there is no mention of `fipsEntry()` in fips.h, or indeed any header files, so that's the heart of the problem.  for better or worse, there are other instances of FIPS POST functions that exist only in .c files.